internal/pkg/slashing/check.go

```diff
@@ -141,24 +140,21 @@ func verifyBlockSignature(ctx context.Context, view FaultStateView, blk block.Bl
 }
 
 // Checks whether at least one of b1, b2 appear in the chain defined by `head`.
-func verifyOneBlockInChain(ctx context.Context, chn chainReader, head block.TipSetKey, b1 block.Block, b2 block.Block, earliest abi.ChainEpoch) error {
-	if chainHasB1, err := chainContainsBlock(ctx, chn, head, b1, earliest); err != nil {
+func verifyOneBlockInChain(ctx context.Context, chn chainReader, head block.TipSetKey, b1 block.Block, b2 block.Block) error {
+	if chainHasB1, err := chainContainsBlock(ctx, chn, head, b1); err != nil {
 		panic(errors.Wrapf(err, "failed to inspect chain")) // This idiosyncratic failure shouldn't go on chain
 	} else if chainHasB1 {
 		return nil
 	}
-	if chainHasB2, err := chainContainsBlock(ctx, chn, head, b2, earliest); err != nil {
+	if chainHasB2, err := chainContainsBlock(ctx, chn, head, b2); err != nil {
 		panic(errors.Wrapf(err, "failed to inspect chain"))
 	} else if chainHasB2 {
 		return nil
 	}
-	return fmt.Errorf("no consensus fault: neither block in chain since %d", earliest)
+	return fmt.Errorf("no consensus fault: neither block in")
 }
 
-func chainContainsBlock(ctx context.Context, chn chainReader, head block.TipSetKey, blk block.Block, earliest abi.ChainEpoch) (bool, error) {
-	if blk.Height < earliest { // Short-circuit
```

Comment:
I'm a little big concerned about this change. We don't have `earliest` anymore, so I don't know what else to do about it.

Reply:
chainContainsBlock was removed from the definition of checking consensus faults.